src/main/webapp/app/shared/layouts/navbar/navbar.component.ts

```diff
@@ -265,6 +265,19 @@ export class NavbarComponent implements OnInit, OnDestroy {
         plagiarism_cases: 'artemisApp.plagiarism.cases.pageTitle',
     };
 
+    studentPathBreadcrumbTranslations = {
+        exams: 'artemisApp.courseOverview.menu.exams',
+        exercises: 'artemisApp.courseOverview.menu.exercises',
+        lectures: 'artemisApp.courseOverview.menu.lectures',
+        learning_goals: 'artemisApp.courseOverview.menu.learningGoals',
+        statistics: 'artemisApp.courseOverview.menu.statistics',
+        discussion: 'artemisApp.metis.discussion.label',
+        code_editor: 'artemisApp.editor.breadCrumbTitle',
+        participate: 'artemisApp.submission.detail.title',
+        live: 'artemisApp.submission.detail.title',
+        courses: 'artemisApp.course.home.title',
```

Comment:
I am not really happy with the inconsistency here :thinking: but its up to you ^^

Reply:
We're just picking the terms we need from the pages we're linking to in a breadcrumb. I don't see a big problem here as this mostly avoids redundant declarations of the same names

Reply:
> We're just picking the terms we need from the pages we're linking to in a breadcrumb. I don't see a big problem here as this mostly avoids redundant declarations of the same names

Yes seems reasonable :+1: